src/components/common/Menu.js

```diff
@@ -1,9 +1,41 @@
 import React from 'react';
+import { func, string } from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { connect } from 'react-redux';
 
-const Menu = () => (
+import { logout } from '../../actions/sessionActions';
+import smilies from '../../img/smilies.svg';
+import profilePic from '../../img/guy.svg';
+import blueCircle from '../../img/blue-circle.svg';
+
+const Menu = ({ logout, user }) => (
   <div className="menu">
     <p className="target-title">TARGET</p>
+    <div className="profile-pic" >
+      <img src={blueCircle} alt="blue" />
+      <img src={profilePic} alt="guy" />
+    </div>
+    <p> { user } </p>
+    <p className="edit-logout">
+      <a className="edit" > <FormattedMessage id="home.edit" /> </a>
```

Comment:
Try not to leave unnecessary spaces:
Turn 
`<a className="edit" > <FormattedMessage id="home.edit" /> </a>` into
`<a className="edit"><FormattedMessage id="home.edit" /></a>`

Check for this in other files too.
My recommendation is that you install the es-lint package for your code editor